.vsconfig

```diff
@@ -0,0 +1,53 @@
+{
+  "version": "1.0",
+  "components": [
+    "Microsoft.VisualStudio.Component.CoreEditor",
+    "Microsoft.VisualStudio.Workload.CoreEditor",
+    "Microsoft.NetCore.Component.Runtime.5.0",
+    "Microsoft.NetCore.Component.Runtime.3.1",
```

Comment:
I think we need both of these runtimes, eh? @Sergio0694 @azchohfi

Reply:
Was playing around with trying to install and uninstall particular components...
The UWP Workload seems to require both those components.
![image](https://user-images.githubusercontent.com/7164552/100290093-7579cf00-2f48-11eb-8486-ccc7940e9687.png)

[ref](https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-community?view=vs-2019&preserve-view=true#universal-windows-platform-development)

Reply:
I was also playing around with creating a solution filter for the .NET Standard projects only, to make life easier for contributors that might only want to contribute to them (for instance, people that are on a Mac or on Linux). Would it make sense to do both things in this PR since it is about a closely related matter? Also about this, should we offer two separate `.vsconfig` files, one of which only targeting the cross platform .NET Standard toolset and without any Windows specific SDK needed? I think just the .NET SDK might be enough for them 🤔

Reply:
If we want to expand this pr, it could also provide some value to connect this file into our CI system. 

If we make it so that the build server only uses the VS comments from this file it could help in many ways. 
1. Makes it easy to add a VS component to the build system.
2. Make it hard to add code to requires a VS component without listing it.